### PR TITLE
Standoff in practice with the cupid command

### PR DESCRIPTION
**Pull request by Randium**
This branch contains functions to successfully manipulate standoffs.

Standoffs are comparable with the death-row, as they are kills that will be executed later on. The difference, however, is that the death-row is emptied at the end of each night. Standoff kills, however, are executed when the "murderer" dies.

Applications of this standoff mechanic are for example the cupid, the zombie, the huntress, the mad scientist, the executioner and even the fortune teller (in which case the fortune apprentice doesn't actually DIE, but an action does need to be made).

Basically, it's a list of actions that need to be taken whenever given users die. And it's really useful.

### CODE REVIEW

File: management/db.py [hunk 1]
```diff
@@ -448,10 +448,39 @@ def delete_freezer(user_id,victim_id):
...
+    # NOT TESTED YET - PLEASE DO
+    """Add a new order to the kill queue.
+
+    Keyword arguments:
+    victim_id -> the id of the victim to be attacked
+    role -> the role of the attacker
+    murderer -> id of the attacker (random attacker if multiple)
+    """
+    data = [victim_id,role,murderer]
+    c.execute("INSERT INTO 'standoff' ('id','victim','role','murderer') VALUES (NULL,?,?,?)",data)
```
Comment (Bentechy66) [inline management/db.py]: Is the `NULL` on `id` here intentional?

File: management/db.py [hunk 2]
```diff
@@ -448,10 +448,39 @@ def delete_freezer(user_id,victim_id):
     conn.commit()
     return True
 
-def add_standoff():
-    """Add a new kill condition to the database."""
-    pass
-    # TODO
+def add_standoff(victim_id,role,murderer):
+    # NOT TESTED YET - PLEASE DO
```
Comment (Bentechy66) [inline management/db.py]: need to test before merge

File: management/db.py [hunk 3]
```diff
@@ -448,10 +448,39 @@ def delete_freezer(user_id,victim_id):
...
+        returntable.append([element[i] for i in range(4)])
+
+    return returntable
+
+def delete_standoff(standoff_id):
+    """Remove a standoff from the database. This is generally not needed, but is used for occasions like the huntress who chooses a new target.
+
+    Keyword arguments:
+    standoff_id -> the database id of the standoff"""
+    c.execute("DELETE FROM 'standoff' WHERE id =?",(standoff_id,))
```
Comment (Bentechy66) [inline management/db.py]: This won't work because you're not setting `id` in the database

File: management/db.py (hunk 1 above)
Comment (Bentechy66) [reply management/db.py]: will setting `NULL` overwrite the autoincrement?

File: test_main.py [hunk 4]
```diff
@@ -106,3 +106,16 @@ def control_freezers():
   assert db.delete_freezer(1,4) == True
   assert db.get_freezers(1) == [(3, 'Booh'), (5, 'Hooker')]
   reset.reset(True)
+
+def mexican():
```
Comment (Bentechy66) [inline test_main.py]: tests won't be run if they don't start with `test_`, I believe

File: management/db.py (hunk 1 above)
Comment (Randium) [reply management/db.py]: In contrary, the autoincrement will overwrite the `NULL`. I used `NULL` as a placeholder, or, I should say, the SQL database manager told me to.

File: management/db.py (hunk 2 above)
Comment (Randium) [reply management/db.py]: Whoops! Forgot to remove that comment. It has been tested, and the evil dr. Travis is currently testing it too.

File: management/db.py (hunk 3 above)
Comment (Randium) [reply management/db.py]: Is this referring to the part about `NULL`? If so, then see that comment.

If you're talking about something else, then let me know.

File: test_main.py (hunk 4 above)
Comment (Randium) [reply test_main.py]: Seems like #67 confirms this. Alright, lemme close that pull request and get this fixed.
(How did you not notice that earlier? There's multiple functions in there that do not start with `test_`!
